### PR TITLE
Header responsiveness

### PR DESCRIPTION
**Pull request by yotamm**
New menu under 1200 with sub-menus.
I did not add any animations.
Please remember the links in the header are not connected anywhere.
I did not remove "contact us" tab in the header, I lowered the paddings on the sides by half.

### CODE REVIEW

File: our-brothers-web/src/app/components/header/header.component.scss [hunk 1]
```diff
@@ -181,34 +143,80 @@
...
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: stretch;
+    padding: 1.5em;
+    box-shadow: -10px 0px 2px 0px rgba(0,160,153,0.75);
+
+    .close-drawer-menu {
+      display: block;
+      color: rgba(0,160,153);
+      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 10 10' enable-background='new 0 0 40 40'%3E%3Cline x1='0' y1='0' x2='10' y2='10' stroke='rgba(0,160,153)' stroke-width='1' stroke-linecap='round' stroke-miterlimit='10'%3E%3C/line%3E%3Cline x1='0' y1='10' x2='10' y2='0' stroke='rgba(0,160,153)' stroke-width='1' stroke-linecap='round' stroke-miterlimit='10'%3E%3C/line%3E%3C/svg%3E");
```
Comment (gilhanan) [inline our-brothers-web/src/app/components/header/header.component.scss]: Can we extract it to SVG file?

File: our-brothers-web/src/app/components/header/header.component.scss [hunk 2]
```diff
@@ -1,92 +1,57 @@
...
     flex: 0 0 auto;
+    display: flex;
+    align-items: center;
+    list-style-type: none;
 
-    button {
-      border: none;
-      background: none;
-      outline: none;
+    & li:last-of-type a {
```
Comment (gilhanan) [inline our-brothers-web/src/app/components/header/header.component.scss]: Please use classes

File: our-brothers-web/src/app/components/drop-down-menu/drop-down-menu.component.ts [hunk 3]
```diff
@@ -8,6 +8,8 @@ import { Component, OnInit, Input, TemplateRef } from '@angular/core';
 export class DropDownMenuComponent implements OnInit {
   @Input() menuContent: TemplateRef<any>;
   @Input() menuAnchor: TemplateRef<any>;
+  @Input() disabled: boolean = false;
+  @Output() click = new EventEmitter<any>();
```
Comment (gilhanan) [inline our-brothers-web/src/app/components/drop-down-menu/drop-down-menu.component.ts]: Do we need it?

Comment (gilhanan) [inline our-brothers-web/src/app/components/drop-down-menu/drop-down-menu.component.ts]: Please remove

File: our-brothers-web/src/app/components/header/header.component.ts [hunk 4]
```diff
@@ -1,16 +1,47 @@
...
+    about: false
+  };
 
-  @Input()
-  public user: User;
+  @Input() public user: User;
 
   constructor(public authService: AuthService) { }
+
+  updateMobileViewState = () => this.isMobileView = window.innerWidth <= 1400;
```
Comment (gilhanan) [inline our-brothers-web/src/app/components/header/header.component.ts]: `@HostListener('window:resize', ['$event'])`. Then we will can rid off from OnInit and OnDestroy

File: our-brothers-web/src/app/components/drop-down-menu/drop-down-menu.component.ts (hunk 3 above)
Comment (yotamm) [reply our-brothers-web/src/app/components/drop-down-menu/drop-down-menu.component.ts]: this helps disable the drop down when we drop below a certain size

File: our-brothers-web/src/app/components/header/header.component.scss (hunk 1 above)
Comment (yotamm) [reply our-brothers-web/src/app/components/header/header.component.scss]: we can but why?

Comment (gilhanan) [reply our-brothers-web/src/app/components/header/header.component.scss]: Better maintenance in the future. But feel free to resolve it as is

File: our-brothers-web/src/app/components/drop-down-menu/drop-down-menu.component.ts (hunk 3 above)
Comment (gilhanan) [reply our-brothers-web/src/app/components/drop-down-menu/drop-down-menu.component.ts]: Cool, I meant the `click` event

File: our-brothers-web/src/app/components/header/header.component.scss (hunk 1 above)
Comment (yotamm) [reply our-brothers-web/src/app/components/header/header.component.scss]: I moved the class to syle.css and changed it to .site-close-icon

File: our-brothers-web/src/app/components/header/header.component.scss (hunk 2 above)
Comment (yotamm) [reply our-brothers-web/src/app/components/header/header.component.scss]: changed

File: our-brothers-web/src/app/components/drop-down-menu/drop-down-menu.component.ts (hunk 3 above)
Comment (yotamm) [reply our-brothers-web/src/app/components/drop-down-menu/drop-down-menu.component.ts]: removed

**Pull request merged**